packages/angular_devkit/build_webpack/src/webpack-dev-server/index.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  BuildEvent,
+  Builder,
+  BuilderConfiguration,
+  BuilderContext,
+} from '@angular-devkit/architect';
+import { Path, getSystemPath, normalize, resolve } from '@angular-devkit/core';
+import { Observable } from 'rxjs';
+import * as webpack from 'webpack';
+import * as WebpackDevServer from 'webpack-dev-server';
+import { WebpackDevServerBuilderSchema } from './schema';
+
+
+export class DevServerBuilder implements Builder<WebpackDevServerBuilderSchema> {
+
+  constructor(public context: BuilderContext) { }
+
+  run(builderConfig: BuilderConfiguration<WebpackDevServerBuilderSchema>): Observable<BuildEvent> {
+    const root = this.context.workspace.root;
+
+    const configPath = resolve(root, normalize(builderConfig.options.webpackConfig));
+    const config = this.loadWebpackConfig(getSystemPath(configPath));
+
+    return this.runWebpackDevServer(config);
+  }
+
+  public loadWebpackConfig(webpackConfigPath: string): webpack.Configuration {
+    return require(webpackConfigPath) as webpack.Configuration;
```

Comment:
`import` instead of `require`?  would make it promise-based and then could be an observable chain above.

Reply:
I'm not sure the semantics would be the same, nor that it would downlevel to `require`. Do you know?

Reply:
It does now but won't necessary in the future.  I, personally, prefer the use of `import` for consistency reasons.

Reply:
Changed to `import`